cli/src/semgrep/meta.py

```diff
@@ -40,6 +40,10 @@ def get_url_from_sstp_url(sstp_url: Optional[str]) -> Optional[str]:
         # let's just pick https
         protocol = "https"
 
+    # if we know that this URL is going to be unclickable, return the original URL
+    if None in [protocol, result.resource, result.owner, result.name]:
+        return sstp_url
```

Comment:
Can someone add a comment somewhere explaining what is an SSTP URL?
I never heard about SSTP.

Reply:
i don't really think i can write a comment about it, i've never heard of it either

all i know is that returning a URL which contains `None` in it is bad behavior

Reply:
ah yeah, perhaps we could add [this link](https://stackoverflow.com/questions/31801271/what-are-the-supported-git-url-formats) which talks about all the different git url formats.